### PR TITLE
Fixed nav gradient to match UX design.

### PR DESCRIPTION
**Pull request by jdhodges**
#638

### CODE REVIEW

File: src/assets/js/chart/nav.js [hunk 1]
```diff
@@ -81,12 +83,21 @@ export default function() {
...
+        .decorate(function(selection) {
+            selection.attr('fill', 'url("#brush-gradient")');
+        });
+
+    var brushLine = fc.series.line()
+        .yValue(function(d) { return d.close; });
 
     var layoutWidth;
 
+    var multi = fc.series.multi()
```
Comment (jleft) [inline src/assets/js/chart/nav.js]: Maybe `multi` => `brushMaskMulti`, so not to be confused with the other multi series?

Comment (djmiley) [inline src/assets/js/chart/nav.js]: Good point.

File: src/assets/js/chart/nav.js [hunk 2]
```diff
@@ -84,13 +86,24 @@ export default function() {
...
     var brushMask = fc.series.area()
-      .xValue(function(d) { return d.date; })
-      .yValue(function(d) { return d.close; })
-      .xScale(maskXScale)
-      .yScale(maskYScale);
+        .xValue(function(d) { return d.date; })
+        .y1Value(function(d) { return d.close; })
+        .y0Value(function(d) { return maskYScale.domain()[0]; })
+        .decorate(function(selection) {
+            selection.attr('fill', 'url("#brush-gradient")');
```
Comment (jleft) [inline src/assets/js/chart/nav.js]: Can we do this on `selection.enter()`?

File: src/assets/styles/navigator.less [hunk 3]
```diff
@@ -60,19 +61,21 @@
       .mask-background {
         fill: @mask-transparent;
       }
-      .area {
-        fill: @mask-opaque;
+
+      .line {
+        stroke-width: 1.2px;
```
Comment (jleft) [inline src/assets/styles/navigator.less]: Just wondering why `1.2px` and not `1px`?

Comment (jdhodges) [inline src/assets/styles/navigator.less]: `1.2px` looked closer to the ux designs, I can change to 1 if you think it'd be better.

Comment (jdhodges) [inline src/assets/styles/navigator.less]: 1px: image

1.2px: image

Which would we prefer?

Comment (jleft) [inline src/assets/styles/navigator.less]: Let's go with 1.2px then :)

**Comment by jleft** [comment]
After https://github.com/ScottLogic/BitFlux/pull/639#discussion_r58212929 is adressed and this is rebased, this looks good to me :+1:

**Comment by jleft** [comment]
Just needs the rebase to resolve conflicts then good to merge, thanks @jdhodges :+1:

**Pull request closed without merging**